### ISSUE TITLE
Doesn't work for reload

### ISSUE THREAD

**Issue by nikhil271995** [hinok/react-router-last-location#18]
If the current route was A and a reload happens on route A and then user moves to route B then last route comes as null instead of A.

**Comment by hinok** [comment]
@nikhil271995 Hey, thanks for reporting the issue.

Probably you meant scenario like this:

1. Visit `/`, `last = null`
2. Visit `/about`, `last = /`
3. Visit `/contact`, `last = /about`
4. Refresh/reload `/contact`, `last = null`

Could you confirm that scenario described by me is what you meant?

**Comment by taverasmisael** [comment]
I can confirm scenario 4. The last location is missing if the page is reloaded. Would be great if you could pass a `persitor` function to the `LastLocationProvider` . In that case, each user could implement its own lastLocation caching strategy.

**Comment by hinok** [comment]
@taverasmisael Yup, I'm happy to add such functionality to the lib. Will try to finish work on this and #17

**Comment by shiro** [comment]
Any update on this?

**Comment by Hervera** [comment]
It still doesn't work on page reload, any update?
